M2Mqtt/MqttClient.cs

```diff
@@ -982,7 +982,7 @@ private bool EnqueueInflight(MqttMsgBase msg, MqttMsgFlow flow)
                     State = state,
                     Flow = flow,
                     Attempt = 0,
-                    Timestamp = Environment.TickCount
+                    Timestamp = (int)Environment.TickCount64
```

Comment:
I know we've been back and force with this one. Still, not *yet* perfect:
* `Timestamp` should be a long so no cast with the TickCount64
* `delta` is fine as an int because it's going to be the difference of 2 long which are in a short range, so it will always be an int
* As a result, then: `delta = (int)(Environment.TickCount64 - msgContext.Timestamp);`

Reply:
Ok, I will make that change. Thanks. (I just based this version on the original code just converting DateTime ticks to an int out the box.

Reply:
Should be good now. Although to note, since this was a public field, hope it is not a breaking change...